docs/sdk/src/meta_contributing.rs

```diff
@@ -134,14 +134,18 @@
 //!
 //! ## How to Develop Locally
 //!
-//! To view the docs specific [`crate`] locally for development, including the correct HTML headers
-//! injected, run:
+//! To expedite documentation development and view _only_ the docs specific [`crate`] locally for
+//! development, including the correct HTML headers injected, run:
 //!
 //! ```sh
+//! # (Fastest) HTML ONLY - Only build the documentation crate, with MINIMAL code compilation.
+//! RUSTDOCFLAGS="--html-in-header $(pwd)/docs/sdk/headers/toc.html" cargo doc -p polkadot-sdk-docs --no-default-features -F min-docs --no-deps --open
+//!
+//! # SDK Docs only - Minimal build - Only build the documentation crate, including code compilation for examples.
 //! SKIP_WASM_BUILD=1 RUSTDOCFLAGS="--html-in-header $(pwd)/docs/sdk/headers/toc.html" cargo doc -p polkadot-sdk-docs --no-deps --open
```

Comment:
This is the command suggested to get the minimal docs build. I did try this again on present master, and what I was experiencing was what I think was very long linking times in the stage of building the full kitchensink runtime and node... I can't seem to reproduce now, so I agree this extra complexity isn't worth it to get fast HTML output to inspect while doing docs dev.